openpype/hosts/maya/plugins/load/load_reference.py

```diff
@@ -242,11 +242,15 @@ def _post_process_rig(self, name, namespace, context, options):
         creator_plugin = get_legacy_creator_by_name(
             self.animation_creator_name
         )
+        animation_subset_name = namespace.replace(
+            name,
+            name.replace('rig', 'animation')
+        )
```

Comment:
Interesting, but also way too hardcoded I'd say. I wonder if there are better ways to find the right subset name for animation instances. @tokejepsen @mkolar any thoughts?

Reply:
@BigRoy Yes I know. You suggested I use something like this, but I haven't found the right place to set animationSubsetName in the options. :)
```
        name = options.get("animationSubsetName", namespace)
        with maintained_selection():
            cmds.select([output, controls] + roots, noExpand=True)
            legacy_create(
                creator_plugin,
                name=name,
                asset=asset,
                options={"useSelection": True},
                data={"dependencies": dependency}
            )
```

Reply:
That's indeed what I recommended for the case you wanted to customize the naming purely from the workfile template system since I believe that system allows to pass custom options to the creators as it builds it. (I believe there's an options field that can take a JSON string with options?)

Reply:
Exactly. I found "Loader Arguments" for this purpose, but in the end, it was not what I was looking for to solve our subset name problem.

Reply:
I'd say this is stuck due to it being hardcoded yes. However, others might have @antirotor @mkolar @jakubjezek001 decent solutions to propose for this?